hazelcast/src/main/java/com/hazelcast/internal/partition/impl/MigrationManager.java

```diff
@@ -1037,22 +986,7 @@ private void processNewPartitionState(PartitionReplica[][] newState) {
         /** Schedules all migrations. */
         private void scheduleMigrations(List<Queue<MigrationInfo>> migrationQs) {
             Version version = nodeEngine.getClusterService().getClusterVersion();
```

Comment:
This version is not used anymore and should be removed. Related import also could be removed.